recipe/meta.yaml

```diff
@@ -24,23 +24,20 @@ source:
   sha256: e16c1214fb7ae1bb60a9224e07b70aa5fd03310226140672ab3b409ddf4ae2a7  # [win]
 
 build:
-  number: 0
+  number: 1
   skip: true  # [osx]
 
 test:
   commands:
     - test -L $PREFIX/lib/libnvToolsExt.so.1                                # [linux]
     - test -L $PREFIX/lib/libnvToolsExt.so.1.0.0                            # [linux]
-    - test -L $PREFIX/targets/{{ target_name }}/lib/libnvToolsExt.so.1      # [linux]
-    - test -f $PREFIX/targets/{{ target_name }}/lib/libnvToolsExt.so.1.0.0  # [linux]
 
 outputs:
   - name: cuda-nvtx
     build:
         skip: true  # [not linux]
     files:
       - lib/libnv*.so.*                       # [linux]
-      - targets/{{ target_name }}/lib/*.so.*  # [linux]
```

Comment:
I moved these to the dev package, but I'm not even sure if these are needed at all.

Reply:
Interesting given we already have these here and other packages have already built against `cuda-nvtx`, am a little hesitant to remove something that packages in user environments may already be depending on

Reply:
Reverted for now